testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -317,6 +317,7 @@ export default PageView.extend({
     function cleanup() {
       evt.target.disabled = false;
       evt.target.classList.remove('state-change');
+      evt.target.style.width = '';
```

Comment:
added these to clear the button width after install/uninstall otherwise the calculated width for the state change persists and you get funky button sizes